modules/accessing-hosts-on-aws.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/accessing-hosts.adoc
+
+[id="accessing-hosts-on-aws_{context}"]
+= Accessing hosts on Amazon Web Services (AWS)
+
+The {product-title} installer does not create any public IP addresses for any of
+the Amazon Elastic Compute Cloud (Amazon EC2) instances that it provisions for
+your {product-title} cluster. In order to be able to SSH to your {product-title}
+hosts, you must follow this procedure.
+
+.Procedure
+
+. Create a security group that allows SSH access into the virtual private cloud
+(VPC) created by the `openshift-install` command.
+
+. Create an Amazon EC2 instance inside the VPC that the installer created or on
+one of the public subnets the installer created.
+
+. Associate a public IP address with the Amazon EC2 instance that you created.
++
+Unlike with the {product-title} installation, you must associate the Amazon EC2
+instance you created with an SSH keypair. It does not matter what operating
+system you choose for this instance, as it will simply serve as an SSH bastion
+to bridge the internet into your {product-title} cluster's VPC.
+
+. Once you provisioned your Amazon EC2 instance and can SSH into it, you must add
+the SSH key that you associated with your {product-title} installation, not the
+key for the bastion instance.
+
+. Run `oc get nodes`, inspect the output, and choose one of the nodes that is a
```

Comment:
Somewhere we want to say that direct SSH access is only recommended for disaster recover.  When the Kubernetes API is responsive, as you're assuming here, folks should run (privileged) pods instead.

Reply:
Thanks! I added a note